app/src/pages/inside/profilePage/constants.js

```diff
@@ -0,0 +1,3 @@
+export const API_KEYS_ROUTE = 'apiKeys';
+export const PROJECT_ASSIGNMENT_ROUTE = 'assignedProjects';
+export const CONFIG_EXAMPLES_ROUTE = 'configExamples';
```

Comment:
Let's move these constants to `common/constants` as we store all sub routes there.
And don't forget to add EPAM signature.